### ISSUE TITLE
Is it possible to send notification using tool such as dunst when some block's value get warning/critical?

### ISSUE THREAD

**Issue by c02y** [greshake/i3status-rust#1667]

**Comment by MaxVerevkin** [comment]
No, this is not currently possible. Can you share your usecase?

**Comment by c02y** [comment]
For instance, when my disk_space block gets critical value and I have to pause the disk IO process or clean some files, but sometimes if I didn't notice the i3status-rust bar, or when I'm in fullscreen mode, so notification will be useful.

For instance when the battery level of gamepad controller from battery block becomes critical (fullscreen game mode), I can change battery before it is dead.

**Comment by cfsmp3** [comment]
I'm not sure we want a status bar to be the center of notifications to be displayed by a different application? I took a look at dunst which (if I got it right) watches from notifications on dbus. Those notifications should be IMHO be generated by a system monitor that is not the status bar.

**Comment by MaxVerevkin** [comment]
So yes, implementing this in not too hard, but it seems to be out of scope for this project. Related: #1148.

**Comment by c02y** [comment]
Since a lot of blocks contains values like warning/critical, maybe implementing such universal command field for warning/critical values can be very helpful, i3status-rust is already checking those values repeatedly in the background, so people don't have to keep another script/service running and checking the values in the background.
